pkg/agent/controller/networkpolicy/l7engine/reconciler.go

```diff
@@ -195,6 +196,14 @@ func convertProtocolHTTP(http *v1beta.HTTPProtocol) string {
 	return strings.Join(keywords, " ")
 }
 
+func convertProtocolTLS(tls *v1beta.TLSProtocol) string {
+	var keywords []string
+	if tls.SNI != "" {
+		keywords = append(keywords, fmt.Sprintf(`tls.sni; content:"%s"; nocase; pcre:"/%s$/;"`, tls.SNI, tls.SNI))
```

Comment:
Why don't use the same way that matches http host and path, but use pcre here? I think we should unify the input format and expectation.

Reply:
I referred to the Suricata examples. I'll update this to unify the input format of sni.